tests/cluster/check_remoteaccount.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2015 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ducktape.errors import TimeoutError
+from tests.ducktape_mock import MockAccount
+from tests.test_utils import find_available_port
+
+from threading import Thread
+
+import SimpleHTTPServer
+import SocketServer
+import threading
+import time
+
+
+class SimpleServer(object):
+    """Helper class which starts a simple server listening on localhost at the specified port
+    """
+    def __init__(self):
+        self.port = find_available_port()
+        self.handler = SimpleHTTPServer.SimpleHTTPRequestHandler
+        self.httpd = SocketServer.TCPServer(("", self.port), self.handler)
+        self.close_signal = threading.Event()
+        self.server_started = False
+
+    def start(self, delay_sec=0.0):
+        """Run the server after specified delay"""
+        def run():
+            end_delay = time.time() + delay_sec
+            while time.time() < end_delay and not self.close_signal.is_set():
```

Comment:
Seems like this is checking things that it shouldn't need to. I thought the point of `Event` is that it doesn't have spurious wakeups so you should just be able to the body of this loop without the loop itself?